CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+## Prerequisites
+
+## Build actors and providers
+
+From the top-level **directory** build with `make`. This should complete
+without errors.
+
+## Configuration
+
+Update paths in file `deploy/env` to match your development environment.
+
+## Running
+
+The script `deploy/run.sh` contains commands to run everything. In the
```

Comment:
I believe `run.sh` should be deprecated, and `run-demo.sh` is the more up to date version

Reply:
ok deleted run.sh and updated references to `run-demo.sh`